remix/app/routes/_app.newsletter.create.test.tsx

```diff
@@ -50,14 +48,7 @@ describe('POST /newsletter/create', () => {
         context: {},
       });
 
-      expect(await result.json()).toMatchObject<Newsletter>({
-        id: newsletter.id,
-        name: expect.any(String),
-        content: expect.any(String),
-        userId: newsletter.userId,
-        createAt: expect.any(String),
-        updateAt: expect.any(String),
-      });
+      expect(await result.status).toBe(302);
```

Comment:
Friendly reminder: We can use `http-status-codes` here 🙈

Reply:
Ohh, I actually have library on the project, but I just follow it 🙊.

I will add it, Thanks for remind me that!

Reply:
I know that feeling 😭

Reply:
Improved here: [07415ce](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/07415cea75c043a849c912f59b3c4428548ba754)